pages/mentors.tsx

```diff
@@ -90,8 +93,15 @@ const App = () => {
     },
   };
 
-  const handleTabChange = (_: React.ChangeEvent<{}>, tab: number) =>
+  const handleTabChange = (_: React.ChangeEvent<{}>, tab: number) => {
     setCurrentTabId(tab);
+    const button = document.querySelector(
+      'input[class="button sui-search-box__submit"]'
+    ) as HTMLButtonElement;
```

Comment:
Please add one line of comment starting with `FIXME: ` and briefly explaining that this is a hack and how it works, so that if it breaks in future library upgrades, we know to look at this code carefully.